### PR TITLE
Product addition(Ecosia)

### PR DESCRIPTION
**Pull request by SIVAH-AKASH**
**Type of pull request:** product addition / product edit / bug fix / other...

**Related issues:** list the related issues here (if applicable)

---

Provide more information about your pull request here. For help, see our contribution guide: http://privacyspy.org/contribute/

### CODE REVIEW

**Review by SIVAH-AKASH — COMMENTED** [review]
I am new to GitHub so I hope I got everything right this time around. :)

File: products/ecosia.toml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+[rubric.law-enforcement]
+value = "unspecified"
+citations = [
+  ""
+]
+
+[rubric.list-collected]
+value = "exhaustively"
+citations = [
+  "\"Temporary storage of IP address, Date and time of enquiry, Difference between time zone and Greenwich Mean Time (GMT), Content of the request (specific webpage), Access status or HTTP status code, Respectively transferred data quantity, Website from which the request comes, Browser, Operating system and its surface, Language and version of the browser software, data in connection with program crashes.\""
```
Comment (ibarakaiev) [inline products/ecosia.toml:73]: Please remove `\"` from the citation.

**Review by ibarakaiev — CHANGES_REQUESTED** [review]
Hi! Thanks for submitting this. I noticed that this submission is a follow-up on your old PR which you closed — in fact, you can update changes in existing pull requests so it's easier to maintain. Before I can add the product to the database, we'd need to implement a few changes. Please go over my comments and resubmit; let me know if you have questions!

**Comment by milesmcc** [comment]
Feel free to reopen if you want to take this further! I've closed it because it has been open for so long—not because we don't want it. (We do.)

**Pull request closed without merging**